src/Modules/OrchardCore.Commerce/Assets/Scripts/discount.js

```diff
@@ -0,0 +1,7 @@
+const prices = Array.from(document.querySelectorAll('.tax-part-gross-price-value, .price-part-price-field-value'));
+
+function strikeOutPrice(price) {
+    price.innerHTML = "<del class=\"text-danger\">" + price.textContent + "</del>";
+}
+
+prices.forEach(price => strikeOutPrice(price));
```

Comment:
According to ESLint:

    7:16  warning  Expected parentheses around arrow function argument  arrow-parens